### PR TITLE
Spexapc 348 - eso integration, vsphere-csi-secret chart

### PR DESCRIPTION
**Pull request by DanielStano**

### CODE REVIEW

File: charts/vsphere-csi-secret/templates/vsphere-csi-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        property: username
+      secretKey: username
+  target:
+    name: vsphere-config-secret
+    template:
+      engineVersion: v2
+      data:
+        csi-vsphere.conf: |
+          [Global]
+          cluster-id = "{{ .Values.clusterName }}-cluster"
```
Comment (sedlak7) [inline charts/vsphere-csi-secret/templates/vsphere-csi-secret.yaml]: @DanielStano , pls could you verify cluster-id 

According to https://github.com/gr8it/conf-lab.gr8it.cloud/tree/main/fleet-applications-common/dev/vsphere-csi

cluster-id = "customer_environment_clustername" 

Thx
P.

**Review by sedlak7 — CHANGES_REQUESTED** [review]

File: charts/vsphere-csi-secret/templates/vsphere-csi-secret.yaml (hunk 1 above)
Comment (DanielStano) [reply charts/vsphere-csi-secret/templates/vsphere-csi-secret.yaml]: so far the customer_environment_clustername is not yet implemented in downstream dev or stable, so i can only work with clusterName as in the chart. an unique cluster-id is needed but so far not implemented. i can either wait and put the work on hold or rework the chart as needed once labeling is implemented in our env

Comment (sedlak7) [reply charts/vsphere-csi-secret/templates/vsphere-csi-secret.yaml]: I believe in power of  helm templating, with Haimdall help you can do it.
https://helm.sh/docs/chart_template_guide/control_structures/

Comment (DanielStano) [reply charts/vsphere-csi-secret/templates/vsphere-csi-secret.yaml]: changed to cluster-id = "{{ .Values.customer }}_{{ .Values.environment }}_{{ .Values.clusterName }}" as per yourself

**Review by sedlak7 — APPROVED** [review]

File: packaged_charts/vsphere-csi-secret-1.0.0.tgz
Comment (pstefka) [inline packaged_charts/vsphere-csi-secret-1.0.0.tgz:1]: Please remove -> 1.0.2 shall exist in the repo + update index

File: packaged_charts/vsphere-csi-secret-1.0.1.tgz
Comment (pstefka) [inline packaged_charts/vsphere-csi-secret-1.0.1.tgz:1]: Please remove -> 1.0.2 shall exist in the repo + update index

**Review by pstefka — APPROVED** [review]
👌

File: packaged_charts/vsphere-csi-secret-1.0.0.tgz
Comment (DanielStano) [reply packaged_charts/vsphere-csi-secret-1.0.0.tgz:1]: done

File: packaged_charts/vsphere-csi-secret-1.0.1.tgz
Comment (DanielStano) [reply packaged_charts/vsphere-csi-secret-1.0.1.tgz:1]: done

**Pull request merged**